models/points.go

```diff
@@ -331,6 +331,10 @@ func ParseTags(buf []byte) Tags {
 	return parseTags(buf, nil)
 }
 
+func ParseTagsWithTags(buf []byte, tags Tags) Tags {
```

Comment:
Could you just rename `parseTags` to `ParseTags` ?

Reply:
Alas, there is already an exported API called `ParseTags`. I'm following the existing convention where `WithTags` is added to the end when the function takes a reusable tags buffer as an argument.

For example, this API does not take an existing `tags` buffer as an argument

```go
func ParseKeyBytes(buf []byte) ([]byte, Tags)
```

this one does

```go
func ParseKeyBytesWithTags(buf []byte, tags Tags) ([]byte, Tags)
```